pyproject.toml

```
@@ -55,6 +55,7 @@ tests = [
     "ipykernel",
     "sparrowpy[fast]",
     "nbmake",
+    "spharpy",
 ]
```

```suggestion
]
examples = [
    "spharpy",
]
```

I woudl suggest to have a dedicated option for examples. ``examples`` need to be added to dev as well

```
dev = ["sparrowpy[deploy,tests,docs,blender,examples]"]
```